src/TodosContext.tsx

```diff
@@ -0,0 +1,224 @@
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
+import { Filter } from './types/Filter';
+import * as todosServices from './api/todos';
+import { TodosContextType } from './types/TodosContextTypes';
+
+export const USER_ID = 11730;
+
+export const TodosContext = React.createContext<TodosContextType>({
+  todos: [],
+  setTodos: () => {},
+  filtredTodos: [],
+  isLoadingTodo: [],
+  errorMessage: '',
+  setErrorMessage: () => {},
+  statusFilter: Filter.ALL,
+  setStatusFilter: () => {},
+  statusResponse: false,
+  setStatusResponse: () => {},
+  tempTodo: null,
+  setTempTodo: () => {},
+  changeErrorMessage: () => {},
+  addTodo: () => {},
+  deleteTodo: () => {},
+  updateTodo: async () => {},
+  activeTodos: 0,
+  hasSomeCompletedTodos: false,
+  handlerClearCompleted: () => {},
+  toggleTodo: () => {},
+  handlerToggleAll: () => {},
+});
+
+type Props = {
+  children: React.ReactNode;
+};
+
+export const TodosProvider: React.FC<Props> = ({ children }) => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoadingTodo, setIsLoadingTodo] = useState<number[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [statusFilter, setStatusFilter] = useState(Filter.ALL);
+  const [statusResponse, setStatusResponse] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isStatusResponse, setIsStatusResponse] = useState(false);
```